atramhasis/static/js/concept.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+    var searchUrl = window.location.search.substr(1);
+    const params = searchUrl.split('&');
+
+    if(params[1] === 'view_tree') {
+      openTreeTab();
+      window.history.pushState('', null, window.location.href.split('?')[0]);
```

Comment:
waarom de keuze van via history te werken? als je nu op back klikt, blijf je in zelfde venster en staat url op 'http://local.onroerenderfgoed.be:6543/conceptschemes/TREES/c/2?&view_tree'
van mij mag die url zo gewoon blijven staan hoor, dan blijft back en forward in browser gewoon werken.